fredericoo/saltplay
# fix: new match not taking seasons into consideration

# 👀 Overview

No more Fridays ruined because of you, pesky bug.

## 🚀 What we have done

- Fixed the POST api endpoint to season to load scores from the relevant season
- Fixed the display on new match to not take other seasons into consideration

## 🏁 How to test

- Have season with games and scores
- create new season
- create new match
- should see 100 pts for everyone
- matches should be worth 10

## ✅ Checklist:
